docs/src/intro.md

```diff
@@ -0,0 +1,57 @@
+# TiSpace Introduction
+
+## Summary
+
+In this project, we will build a system that supports developers to use the K8s Pod as a virtual machine.
+
+The system has the following features:
+
+- Flexible and user-friendly resource management platform
+- Highly available and schedulable disk services
+- Secure Container Runtime for Lightweight Virtual Machines
+- Delivers consistent performance as standard Linux containers
+
+
+## Motivation
+
+At PingCAP, our developers use a lot of virtual machines for our development and testing work. However, these virtual machines have poor performance and they are heavily overdraft. In addition, at PingCAP we do not have a good management platform to distribute and manage these VMs for developers. Developers manage and request these machine resources by asking each other. 
+
+To optimize these inefficient processes, we decided to develop a management platform to efficiently manage and allocate machine resources.
+
+## Detailed design
+
+The whole system uses [K8s] as the infrastructure, and deploys [ceph] clusters in [K8s] for disk high availability, and replaces the [K8s] container layer with [kata] to provide good virtualization capability.
+
+<!-- TODO: Add architecture diagram -->
+
+### K8s
+
+We have deployed [K8s] clusters on bare machine and will rely on [K8s] to provide the scheduling capabilities for the entire system. With this, users only need to provide resource requests when creating instances, and we can use [K8s] scheduling capabilities to create the corresponding Pod resources for users.
```

Comment:
We may need to explain how to make modifications on rootfs persistent (like "mount persistent volume as container rootfs"). But I'm not sure where to place it, K8s or Ceph?

Reply:
Added.